index.js

```diff
@@ -38,11 +38,15 @@ export function trimLines(value) {
  */
 function trimLine(value, start, end) {
   if (!start) {
-    value = value.replace(/^[ \t]+/, '')
+    const regex = /^[ \t]+/
+    value = value.replace(regex, '')
```

Comment:
Is this really needed?
If it is needed, could `regex` be moved to the global scope and called `whitespaceInitial` or so?